### PR TITLE
fix headers in EventFilter pkgs

### PR DESCRIPTION
**Pull request by mrodozov**
Add missing headers for passing check-headers scram check

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: EventFilter/L1TRawToDigi/interface/PackerTokens.h [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #ifndef EventFilter_L1TRawToDigi_PackerTokens_h
 #define EventFilter_L1TRawToDigi_PackerTokens_h
 
+#include "FWCore/Utilities/interface/EDGetToken.h"
```
Comment (smuzaffar) [inline EventFilter/L1TRawToDigi/interface/PackerTokens.h]: this does not look right.

File: EventFilter/GEMRawToDigi/interface/VFATdata.h [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #ifndef EventFilter_GEMRawToDigi_VFATdata_h
 #define EventFilter_GEMRawToDigi_VFATdata_h
+#include "stdint.h"
```
Comment (slava77) [inline EventFilter/GEMRawToDigi/interface/VFATdata.h]: system includes are typically in the brackets ```<stdint.h>```

Comment (mrodozov) [reply EventFilter/GEMRawToDigi/interface/VFATdata.h]: fixed here https://github.com/cms-sw/cmssw/pull/23189. I'd separate this pr into pieces

**Comment by perrotta** [comment]
-1
Since #23189, #23190 have been merged (the ones to be signed by reco), and #23188 has been submitted, I expect this PR is not needed any more, and can be closed therefore.
Alternatively, remove what already merged/submitted and go ahead here with the rest

**Comment by mrodozov** [comment]
@perrotta it's not. Closing this in favor of https://github.com/cms-sw/cmssw/pull/23239

**Pull request closed without merging**